projects/canopy/src/lib/forms/radio/radio-button.component.ts

```diff
@@ -82,6 +86,17 @@ export class LgRadioButtonComponent implements OnInit {
     return this.errorState.isControlInvalid(this.control, this.controlContainer);
   }
 
+  _hintElement: LgHintComponent;
```

Comment:
@Zimann could you add some unit tests please?

Reply:
Added test for this change 😉 .